components/base/client_test.go

```diff
@@ -221,16 +213,16 @@ func TestClient(t *testing.T) {
 		test.That(t, err, test.ShouldBeNil)
 
 		err = failingBaseClient.MoveStraight(context.Background(), 42, 42.0, nil)
-		test.That(t, err.Error(), test.ShouldContainSubstring, errMsgMoveStraight)
+		test.That(t, err.Error(), test.ShouldContainSubstring, errMoveStraight.Error())
 
 		err = failingBaseClient.Spin(context.Background(), 42.0, 42.0, nil)
-		test.That(t, err.Error(), test.ShouldContainSubstring, errMsgSpin)
+		test.That(t, err.Error(), test.ShouldContainSubstring, errSpinFailed.Error())
 
 		_, err = failingBaseClient.Properties(context.Background(), nil)
-		test.That(t, err.Error(), test.ShouldContainSubstring, errMsgProperties)
+		test.That(t, err.Error(), test.ShouldContainSubstring, errPropertiesFailed.Error())
 
 		err = failingBaseClient.Stop(context.Background(), nil)
-		test.That(t, err.Error(), test.ShouldContainSubstring, errMsgStop)
+		test.That(t, err.Error(), test.ShouldContainSubstring, errStopFailed.Error())
```

Comment:
If you change these checks to `test.That(t, err, test.ShouldBeError, errStopFailed)`, does it still work? That's more of a hassle to change from the current implementation, but might be easier to understand by other people in the future.

This comment applies to all lines changed in this file, and all lines changed in about half the other files, too.